ceto/ais_adapter.py

```diff
@@ -177,7 +181,7 @@ def _guesstimate_design_speed(length: float, imo_ship_type: str, speed: float) -
     design_speed = ms_to_knots(
         (froude_number * math.sqrt(g * length)) / block_coefficient
     )
-    return speed if speed > design_speed else design_speed
+    return speed if 0.5 * design_speed <= speed <= 1.5 * design_speed else design_speed
```

Comment:
This guesstimate seems to me inappropriate. The design speed and design draft play a big role in the estimation of the engine load. If the design speed matches the current speed, then the engine load will correspond to the MCR. With this guesstimate that will be the case for a wide range of speeds. Say that the design speed is 20 kn. Then the engine power will be equivalent to the MCR between 10 kn and 30 kn. The previous assumption `return speed if speed > design_speed else design_speed` seemed more reasonable. It addressed the case where we underestimated the design speed and prevented engine load beyond 100%.